hasher-matcher-actioner/hmalib/storage/dynamostore.py

```diff
@@ -1,19 +1,22 @@
 from mypy_boto3_dynamodb.service_resource import Table
 
-class _IDynamoDBItem():
+
+class _IDynamoDBItem:
     """
     Clowny type as an interfaces. True interfaces in python are nightmares.
     """
+
     def to_dynamodb_item(self) -> dict:
         pass
 
-class HashStore:
+
+class DynamoStore:
     """
-    Stores all kinds of hashes into a dynamodb.
+    Stores all kinds of item into a dynamodb.
     """
 
     def __init__(self, table: Table):
         self._table = table
 
-    def add_hash(self, record: _IDynamoDBItem):
+    def add_item(self, record: _IDynamoDBItem):
         self._table.put_item(Item=record.to_dynamodb_item())
```

Comment:
I'm not sure using this class directly should be common, since you generally want the thing you put into the store to be typed to what the table is doing. One advantage of using a class dedicated to the table is that you know what you are interacting with, and can tailor the interfaces to it (for example, instead of _IDynamoDBItem, it can take the appropriate item).

Reply:
Discussed offline, the model is:

Record objects contains all the details for how to store and iteract with it, this class is used for dependency management.

David was expecting an interface to a specific table, and data objects that didn't know their storage details for some reason.

Reply:
Removed 'store' class altogether!